src/contract/sent_transaction.ts

```diff
@@ -53,61 +49,38 @@ export class SentTransaction<T> {
   };
 
   constructor(
-    public signTransaction: ClientOptions["signTransaction"],
-    public assembled: AssembledTransaction<T>,
+    public options: SentTransactionOptions<T>,
```

Comment:
Oh wait this is a breaking change, right? :no_entry:

Reply:
Yeah, good call, we need to add that to the `breaking changes` section of the changelog

Reply:
Is there any way we can avoid it? It's too late for breaking changes since we already tagged off a non-RC build.

Reply:
Sure, let me revert that change, it's going to make for some silly code but that's better than a breaking change